internal/conditions/conditions.go

```diff
@@ -57,6 +58,7 @@ const (
 )
 
 var commonMessages = map[string]string{
+	ReasonEndpointInvalid:         "Endpoint invalid: %s",
```

Comment:
Endpoint invalid sounds too generic. Maybe let's have separate messages for LogPipeline (HTTP output host invalid) and MetricPipeline & Trace Pipeline (OTLP output endpoint invalid)